lib/helpers/get-spatial-reference.js

```diff
@@ -2,10 +2,10 @@ const _ = require('lodash')
 const getCollectionCrs = require('./get-collection-crs')
 const normalizeSpatialReference = require('./normalize-spatial-reference')
 
-function getSpatialReference (geojson, {
-  inputCrs,
-  sourceSR
-}) {
+function getSpatialReference (geojson, props) {
+  if (!props && _.isEmpty(geojson)) return
+  const inputCrs = props ? props.inputCrs : undefined
+  const sourceSR = props ? props.sourceSR : undefined
```

Comment:
Any particular reason why you changed this?  Maybe because you're not sure if the second argument is going to be defined?  Can you just give it a default value and do:

```js
function getSpatialReference (geojson, {
  inputCrs,
  sourceSR
} = {}) {
   if (!inputCrs && !sourceSR && _.isEmpty(geojson)) return
```
FYI, if you didn't want to do the default value in the function definition, you still could avoid the ternarys:
```js
const { inputCrs, sourceSR } = props || {}
```